build/configs/rtl8730e/rtl8730e_make_bin.sh

```diff
@@ -19,6 +19,7 @@
 THIS_PATH=`test -d ${0%/*} && cd ${0%/*}; pwd`
 TOP_PATH=${THIS_PATH}/../../..
 CONFIG=${TOP_PATH}/os/.config
+RESOURCE=${TOP_PATH}/resource
```

Comment:
Hello @jeongarmy , as discussed, after you complete the separation for chipset specific files in resource directory, I will change this file path to` /resource/rtl8730e/...bin`
Thank you~